raiden/raiden_service.py

```diff
@@ -308,16 +312,27 @@ def stop(self):
         self.alarm.stop_async()
         self.protocol.stop_and_wait()
 
+        timeout_s = 2
```

Comment:
Please refrain from unnecessary single letter abbreviations,  `timeout_seconds` would be just fine here.

Reply:
IMO this also should be made into a config in [DEFAULT_CONFIG](https://github.com/raiden-network/raiden/blob/08b8c819db1ac47ba4a9fe7378521751cd35f3be/raiden/app.py#L28), under a new key `ethereum_node` or something like it.

Reply:
should we add a command line flag for changing this as well?